app/src/main/java/org/jellyfin/androidtv/ui/startup/EditServerScreen.kt

```diff
@@ -0,0 +1,51 @@
+package org.jellyfin.androidtv.ui.startup
+
+import androidx.core.os.bundleOf
+import org.jellyfin.androidtv.R
+import org.jellyfin.androidtv.auth.AuthenticationStore
+import org.jellyfin.androidtv.ui.preference.dsl.OptionsFragment
+import org.jellyfin.androidtv.ui.preference.dsl.optionsScreen
+import org.koin.android.ext.android.get
+import java.util.*
+
+class EditServerScreen : OptionsFragment() {
+
+	override val screen by optionsScreen {
+		val serverUUID = requireArguments().get(ARG_SERVER_UUID)
+
+		if (serverUUID !is UUID)
+			return@optionsScreen
+
+		val authenticationStore = get<AuthenticationStore>()
+
+		val server = authenticationStore.getServer(serverUUID) ?: return@optionsScreen
+
+		title = context?.getString(R.string.lbl_manage_users, server.name)
+
+		for (entry in server.users) {
+			link {
+				title = entry.value.name
+				icon = R.drawable.ic_user
+				withFragment<RemoveServerUserScreen>(bundleOf(
+						RemoveServerUserScreen.ARG_REMOVE_USER to true,
+						RemoveServerUserScreen.ARG_SERVER_UUID to serverUUID,
+						RemoveServerUserScreen.ARG_USER_UUID to entry.key
+				))
+			}
+		}
+
+		link {
+			setTitle(R.string.lbl_remove_server)
```

Comment:
I'd use a trash icon here. Also this one could maybe use a description like "This also removes all users"